docs/experiment/guides/troubleshooting/variant-jumping.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Variant Jumping
+description: Learn to troubleshoot variant jumping; description, causes, and solutions.
+---
+
+Variant jumping describes when a user is exposed two or more variants for a single flag or experiment. Variant jumping may occur normally or abnormally, for various reasons. Variant jumping above a certain threshold may be cause for concern in being able to trust an analysis.
+
+## Debugging
+
+The best way to debug variant jumping is by identifying a user that has jumped variants, and analyzing their user timeline. If you're using remote evaluation, you should also check the Assignment event to identify assignment vs exposure discrepancies.
+
+!!!info "To find a user who has jumped variants:"
+
+    1. Go to the monitor tab of your experiment
+    2. Find the variant jumping charts, and click "Open in Analytics"
+    3. Select the bar for users who jumped variants and click "View User Streams" (you may need to use uniques instead of a formula)
+
+    You may find it helpful to "only show" assignment and exposure events in the user timeline.
+
+Important things to note when debugging a user timeline include:
+
+1. Did you introduce any [targeting changes](#targeting-changes) while your flag/experiment was active? Could the timing of this change have effected the variant assigned to this user?
```

Comment:
Maybe add that "You can see what changes were made to the flag/experiment by clicking on the "Version X" button to see the version history"